Source/Extensions/Blazorise.Charts/Options/ChartArcElements.cs

```diff
@@ -0,0 +1,54 @@
+﻿#region Using directives
+using System.Text.Json.Serialization;
+#endregion
+
+namespace Blazorise.Charts
+{
+    /// <summary>
+    /// Arcs are used in the polar area, doughnut and pie charts.
+    /// </summary>
+    public class ChartArcElements
```

Comment:
Can we use records? Are we forgetting about them? Or is there any limitation?

Reply:
I was thinking about that. But it would not make sense at the moment. Because even we have records for this extension, it still uses regular classes from the core Chart project. So, the equality operator would not work as expected. I plan to convert all options to records in v2.0.